.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+
+# License
+###### The [license](https://github.com/mtorials/hugo-mt32/blob/master/CONTRIBUTING.md)
+* [ ] I agree with the license
+* [ ] I do not agree with the license
+
+
+# Information for Contributing
+
+* [ ]  Change Code or Text
+* [ ]  Add new Code or Text
+
+# Pull Request Description
+
+* [ ]  New Blog entry
+* [ ]  Correction Blog entry
+* [ ]  Other
+
+# Description
+
+<!-- Please describe briefly and in summary your pull request -->
+
+## Information
+<!-- Other additional information? -->
+
+# Contribution
+
+Please read our [Contributing](https://github.com/mtorials/hugo-mt32/blob/master/CONTRIBUTING.md) before submitting a pull_request.
+With your pull_request you agree that our [license](https://creativecommons.org/licenses/by-nc-sa/4.0/) is subject to. We ask you to read it before you make the pull_request. 
```

Comment:
you agree to out license. pull request without the _

Reply:
I've remove the line for you.